warehouse/api.go

```diff
@@ -272,9 +283,15 @@ func (uploadsReq *UploadsReqT) TriggerWhUploads() (err error) {
 		}
 	}
 	if (pendingUploadCount + pendingStagingFileCount) == int64(0) {
-		return fmt.Errorf("No pending jobs to trigger for destination : %s", uploadsReq.DestinationID)
+		err = nil
+		response = &proto.TriggerWhUploadsResponse{
+			Message:    fmt.Sprintf("No pending jobs to trigger for destination : %s", uploadsReq.DestinationID),
```

Comment:
Lets change this to `No pending events to sync for this destination`

Reply:
Done.